zeroconf/__init__.py

```diff
@@ -503,6 +503,10 @@ def reset_ttl(self, other: 'DNSRecord') -> None:
         self.created = other.created
         self.ttl = other.ttl
 
+        """ reset expiration times """
```

Comment:
This comment seems redundant, the code already expresses that.

Reply:
Agreed, removed in latest commit